### PR TITLE
updated SKU tags for unlisted products

### PR DESCRIPTION
**Pull request by kateryna-lukina**
## What This PR Changes
- added SKUs to all unlisted products' product.md files

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/_unlisted/kits/junior-certification-bundle/product.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 title: Junior Certification Bundle
 certifications: [CE, UKCA]
+sku: [AKX00043]
```
Comment (jhansson-ard) [inline content/hardware/_unlisted/kits/junior-certification-bundle/product.md:4]: Should this also have `AAX00009`?

File: content/hardware/_unlisted/kits/starter-kit/product.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
 title: Starter Kit
 certifications: [CE, UKCA]
+sku: [K000007, K010007, K020007, K030007, K040007, K050007, K060007, K090007, K110007, K120007]
```
Comment (jhansson-ard) [inline content/hardware/_unlisted/kits/starter-kit/product.md:4]: Missing `K130007`?

Comment (kateryna-lukina) [reply content/hardware/_unlisted/kits/starter-kit/product.md:4]: yes but I haven't created a declaration for it yet :/ I am not sure if it is already released?

Comment (jhansson-ard) [reply content/hardware/_unlisted/kits/starter-kit/product.md:4]: ah I see! Ok, then we'll skip that one!

**Review by jhansson-ard — APPROVED** [review]

**Pull request merged**